src/pkcs11/framework-pkcs15.c

```diff
@@ -1208,6 +1207,13 @@ pkcs15_init_slot(struct sc_pkcs15_card *p15card, struct sc_pkcs11_slot *slot,
 						p15card->tokeninfo ? p15card->tokeninfo->label : "",
 						32);
 			}
+			/* Some applications (NSS) do not like the colons in the
+			 * TOKEN_INFO label so replace them here */
+			for (int i = 0; i < 32; i++) {
+				if (slot->token_info.label[i] == ':') {
+					slot->token_info.label[i] = '_';
```

Comment:
I think underscores are very unpleasant in user messages. Could we maybe agree on replacing that just with a space (my preferred option) or with `.`?